### PR TITLE
Reduce payload of getAll tenant configuration call to only contain Tenant and Status

### PR DESCRIPTION
**Pull request by ftdholbrook**
This is WIP for the broader re-factor of the status service interactions.  It provides some decent quality of life improvements to the list view tenant and sandbox pages that I think it is worth merging now.

- Have TenantAdministrationStatusContext hold only Tenant instead of full tenant configuration
- Only populate tenant and status on tenant configuration

### CODE REVIEW

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java [hunk 1]
```diff
@@ -153,58 +155,6 @@ public boolean exists(final String key) {
...
-
-        //we aren't storing this recreating based off convention
-        if (isSandbox) {
-            //sandbox only
-            tenantConfigurationBuilder.parentTenantKey(parentTenantKey(tenantKey));
-        }
-
-        return Optional.of(tenantConfigurationBuilder.build());
-    }
-
```
Comment (ftdholbrook) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: unrelated dead code removal

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java [hunk 2]
```diff
@@ -226,11 +176,15 @@ private String getLanguageCode(final List<String> uiLanguages) {
...
-                .keySet()
+                .entrySet()
                 .stream()
-                .map(tenantKey -> ImmutablePair.of(tenantKey, findActiveTenantConfiguration(tenantKey)))
-                .filter(pair -> pair.getRight().isPresent())
-                .collect(Collectors.toMap(ImmutablePair::getLeft, pair -> pair.getRight().get()));
+                .map(entry -> ImmutablePair.of(entry.getKey(), TenantConfiguration.builder()
+                        .tenant(entry.getValue())
+                        .administrationStatus(TenantAdministrationStatusContext.builder()
+                                .tenantAdministrationStatus(ACTIVE) //TODO: temporary - next move to activeTenantsByType
```
Comment (ftdholbrook) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: this is WIP future version get all will be full driven from status service

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java [hunk 3]
```diff
@@ -226,11 +176,15 @@ private String getLanguageCode(final List<String> uiLanguages) {
...
+                                .build())
+                        .build()))
+                .collect(Collectors.toMap(ImmutablePair::getLeft, ImmutablePair::getRight));
```
Comment (malaffoon) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: This feels like this could be simpler:
```
    .stream()
    .collect(Collectors.toMap(e -> e.getKey(), e -> TenantConfiguration.builder()...));
```

File: admin-service/src/test/java/org/opentestsystem/rdw/admin/multitenant/web/TenantControllerIT.java [hunk 4]
```diff
@@ -2,14 +2,12 @@
```
Comment (malaffoon) [inline admin-service/src/test/java/org/opentestsystem/rdw/admin/multitenant/web/TenantControllerIT.java]: 👍
